packages/grid/_modules_/grid/components/cell/GridTreeDataGroupingCell.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { unstable_composeClasses as composeClasses } from '@mui/core';
```

Comment:
`@mui/core` is not a dependency.

```suggestion
import { unstable_composeClasses as composeClasses } from '@mui/material';
```